src/motor-operations.h

```diff
@@ -62,22 +74,31 @@ class MotorOperations {  // Rename SegmentQueue ?
 
   // Wait, until all elements in the ring-buffer are consumed.
   virtual void WaitQueueEmpty() = 0;
+
+  // Get the absolute position and auxes status currently
+  // executed by the backend.
+  virtual void GetRealtimeStatus(PhysicalStatus *status) = 0;
```

Comment:
We have several implementations (in tests or other simulations), that can't provide the PhysicalStatus and dont' update the value. Maybe we should have this method return a boolean indicating if the value was updated.